src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/AddPersonnelSideSheet/AddPersonnelForm/components/AddPersonnelFormDisciplinesDropDown.tsx

```diff
@@ -37,19 +38,17 @@ const AddPersonnelFormDisciplinesDropDown: React.FC<PersonnelFormDisciplinesDrop
 
     const onSelect = React.useCallback(
         (newValue: SearchableDropdownOption) => {
-            const changedPerson = { ...item };
-            changedPerson.disciplines = [{ name: newValue.title }];
-            onChange(changedPerson);
+            onChange({ ...item, disciplines: [{ name: newValue.title }] });
         },
-        [item, onChange]
+        [item]
```

Comment:
This should still have a dependency on onChange. If not the onSelect callback could be bound to older versions of onChange